includes/header.php

```diff
@@ -4,6 +4,45 @@
     <meta charset="utf-8">
     <title>Treat Street</title>
     <link href="/styleguide/styles.css" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/b25da52a36.js" crossorigin="anonymous"></script>
 </head>
 <body>
+    <header>
+        <div class="row">
+            <h2 style="background-color: #2E9E47" class="three columns">Treat Street</h2><!--replace with image-->
+
+            <form action="/" method="GET" class="eight columns">
+                <input id="search" type="text" placeholder="Search">
+                <input id="submit" type="submit" value="Search">
+            </form>
+
+            <span class="one column">
+                <a href="/"><i class="fas fa-cart-plus"></i></a>
+                <a href="/"><i class="fas fa-question-circle"></i></a>
+                <a href="/"><i class="fas fa-user-circle"></i></a>
+            </span>
+        </div>
+
+        <div class="row">
+            <p class="twelve columns">Your One-Stop Shop For Your Furry Friends!</p>
+        </div>
+
+        <nav class="row">
+            <a href="/" class="two columns">Snacks</a>
+            <a href="/" class="two columns">Dress Up</a>
+            <a href="/" class="two columns">Play Time</a>
+            <a href="/" class="two columns">Health</a>
+            <a href="/" class="two columns">Just For You</a>
+            <a href="/" class="two columns">Adopt</a>
+        </nav>
+
+
+    </header>
 <!-- END HEADER -->
+
+<?php
+//git push origin header
+
+//go to fork > plus sign > comment > check mark > three dots > push
+?>
+
```

Comment:
Please remove these comments. Anything that you don't want all other groups to see should be removed before submitting the PR :)

Reply:
Done - Shanen

Reply:
These comments have been removed